internal/setup/setup.go

```diff
@@ -27,18 +27,18 @@ import (
 )
 
 const (
-	// ModeCreate is the primary argument to the setup tools container when the task is creating objects
+	// ModeCreate is the primary argument to the setup tools container when the task is creating objects.
```

Comment:
This is going to make the linter so happy.

Reply:
> This is going to make the linter so happy.

This was done by the linter to make itself happy :D